packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -224,3 +225,9 @@ export const ProgrammingLanguageQuestion: SingleSelectQuestion = {
     return "Select a programming language.";
   },
 };
+
+export const GrantPermissionQuestion: TextInputQuestion = {
```

Comment:
We should also consider to ask user to select environment

Reply:
Fixed. Thanks.